### PR TITLE
match @{ and @}

### LINKED ISSUE #4057: Incorrect grouping commands
## Issue Details

In a lot of files we see that there are spare `@}` commands, i.e. `@}` commands without preceding `@{` command.
Until now doxygen does not signal this inconsistency, but the proposed patch https://github.com/doxygen/doxygen/pull/7105 does also handle this inconsistency.
Looking at a few places it looks like that always /often after a `\ingroup` command the `@{` is missing.

The total list of problems I found is: [balance.txt](https://github.com/CGAL/cgal/files/3362823/balance.txt)

Irrespective of the mentioned proposed fix in doxygen I think the problem should be solved.

### PR DESCRIPTION
**Pull request by sloriot**
Should fixes #4057

### CODE REVIEW

**Comment by maxGimeno** [comment]
According to Travis, that's not working at all :/

**Comment by sloriot** [comment]
Quite surprising, I've went again through the diff and everything seems to be happening in comments. I'll have a more careful look later.

File: Polygon/include/CGAL/Polygon_2.h [hunk 1]
```diff
@@ -526,7 +526,7 @@ class Polygon_2 {
     Traits_P traits;
 };
 
-/// @} // polygon_2
+} // CGAL
```
Comment (maxGimeno) [inline Polygon/include/CGAL/Polygon_2.h]: I think the problem lies here. That `{` shoulf be commented.

**Comment by lrineau** [comment]
The documentation testsuite is very yellow:
https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.0-Ic-79/index.html

**Comment by sloriot** [comment]
My detection might be too simplistic and it seems that I missed a lot of them. I'll compile doxygen master later and fix it.

**Comment by lrineau** [comment]
Good. The errors number is now back to 0: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.0-Ic-81/index.html

**Comment by lrineau** [comment]
Now that the branch is tested... have you considered backporting it to release branches? Or maybe we want to support only Doxygen-1.8.13 for release branches.

**Comment by lrineau** [comment]
And also note that the group counting in Doxygen is not yet stable. There is a new pull-request: https://github.com/doxygen/doxygen/pull/7115.

**Comment by sloriot** [comment]
This was too painful, I'm not doing the backport.

**Comment by lrineau** [comment]
[**@sloriot**](https://github.com/sloriot) commented on [Jul 11, 2019, 9:35 AM GMT+2](https://github.com/CGAL/cgal/pull/4060#issuecomment-510367463 "2019-07-11T07:35:21Z - Replied by Github Reply Comments"):
> This was too painful, I'm not doing the backport.

I understand. Let's state that for release branches we only support Doxygen-1.8.13.

**Comment by albert-github** [comment]
Backporting is not necessary as the ouput didn't really change and in 1.8.13 there were, unfortunately, no error messages.
The doxygen pul requests are of no influence on CGAL (at the moment):
- doxygen/doxygen#7115  is in case an error is present the counting was not reset, so it gave just too many warnings
- https://github.com/doxygen/doxygen/pull/7116 is only of influence when `//` etc. normal comments are used insread of `///` doxygen comments
- https://github.com/doxygen/doxygen/pull/7122 might be of influence, but as far as I case gather from the above comments the files involved:
  ```
  GraphicsView/include/CGAL/Qt/camera.h
  GraphicsView/include/CGAL/Qt/constraint.h
  GraphicsView/include/CGAL/Qt/frame.h
  GraphicsView/include/CGAL/Qt/keyFrameInterpolator.h
  GraphicsView/include/CGAL/Qt/manipulatedCameraFrame.h
  GraphicsView/include/CGAL/Qt/manipulatedFrame.h
  GraphicsView/include/CGAL/Qt/mouseGrabber.h
  GraphicsView/include/CGAL/Qt/qglviewer.h
  GraphicsView/include/CGAL/Qt/quaternion.h
  GraphicsView/include/CGAL/Qt/vec.h
  Periodic_2_triangulation_2/include/CGAL/Periodic_2_triangulation_2.h
  Periodic_3_triangulation_3/include/CGAL/Periodic_3_Delaunay_triangulation_3.h
  Periodic_3_triangulation_3/include/CGAL/Periodic_3_regular_triangulation_3.h
  Periodic_3_triangulation_3/include/CGAL/Periodic_3_triangulation_3.h
  Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPCorefinementVisitor.h
  Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPDistanceTraits.h
  Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPSelfIntersectionTraits.h
  QP_solver/doc/QP_solver/fig_src/documentation/algorithm.sty
  Spatial_searching/benchmark/Spatial_searching/include/nanoflann.hpp
  Surface_mesh_approximation/doc/Surface_mesh_approximation/Concepts/ErrorMetricProxy.h
  Testsuite/test/parse-ctest-dashboard-xml.py
  ```
  are not used in the CGAL documentation.

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-87.shtml

**Pull request merged**